CONTRIBUTING.md

```diff
@@ -1,159 +1,42 @@
 # Contributing to Juicebox
 
-If you're interested in contributing ideas or code to Juicebox, you're in the
-right place!
+If you're interested in contributing ideas or code to Juicebox, you're in the right place!
 
 ## Development
 
-Check out the [README](README.md#usage) for instructions on running the app in
-development.
+Read the [development guidelines](doc/development.md) for instructions on running the app in development.
 
 ## Finding something to work on
 
-Start with issues labelled
-[`good first issue`](https://github.com/jbx-protocol/juice-juicehouse/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+Start with issues labelled [`good first issue`](https://github.com/jbx-protocol/juice-juicehouse/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
 
 ## Getting your pull request reviewed, approved, and merged
 
-Create a pull request (PR) that targets the `main` branch. A live Fleek preview
-will be automatically deployed for each PR.
+Create a pull request (PR) that targets the `main` branch. A live Fleek preview will be automatically deployed for each PR.
 
-When your PR has met the [#approval guidelines](approval-guidelines) and is
-ready for review, `@mention` a [codeowner](.github/CODEOWNERS) and ask for a
-review.
+When your PR has met the [approval guidelines](#approval-guidelines) and is ready for review, `@mention` a [codeowner](.github/CODEOWNERS) and ask for a review.
 
 ### Git workflow
 
-We opt for a **rebase policy** where the repository history is kept flat and
-clean. When a feature branch's development is complete, rebase/squash all the
-work down to the minimum number of meaningful commits.
+`main` is our main branch.
 
-While the work is still in progress and a feature branch needs to be brought up
-to date with the upstream target branch, use rebase – as opposed to pull or
-merge – to avoid polluting the commit history with spurious merges.
-[Learn more](https://www.atlassian.com/git/articles/git-team-workflows-merge-or-rebase)
-about the differences between merge and rebase Git workflows.
+Developers should create branches from the `main` branch. Once the code is ready, create a pull request that targets the `main` branch.
```

Comment:
I think it could be worth adding another sentence somewhere around here about dealing with translation files. Maybe:

If you have made any **user-facing string** changes, ensure you have updated the **translation** (`.po`) files. Learn more<link to internationalization>

Reply:
Good call, added a section above this.